app/views/agents/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "New agent" -%>
```

Comment:
How about `Create Agent` to have the title state what action can be made on the page?